packages/ERTP/src/issuer.js

```diff
@@ -14,11 +14,9 @@ import { makeInterface, ERTPKind } from './interfaces';
 import './types';
 
 /**
- * @param {string} allegedName
- * @param {AmountMathKind} [amountMathKind=MathKind.NAT]
- * @returns {IssuerKit}
+ * @type {MakeIssuerKit}
  */
-function makeIssuerKit(allegedName, amountMathKind = MathKind.NAT) {
+function makeIssuerKit(allegedName, amountMathKind = MathKind.NAT, decimals) {
```

Comment:
This is weirdly special case, to include a parameter whose name makes it specific to one particular choice of the previous parameter, `nat`, when that previous parameter represents an (eventually) extensible set of choices. What happens when we have another math kind whose meaning is conditioned on other parameters?

```suggestion
function makeIssuerKit(allegedName, amountMathKind = MathKind.NAT, options = {}) {
  const { decimalPlaces } = options;
```

The meaning of `decimal` is unclear. When I talk about this verbally, I say "decimal places".

Reply:
Yeah, I agree. Ok, I will adopt this! Instead of `options`, what if I call this `displayData` or `displayInfo`? If later, we want to expand it, we can do so without causing incompatibilities since the name is only at the documentation level.